multi_project/api/src/main/java/multi_project/api/App.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package multi_project.api;
+
+
+import multi_project.core.Utils;
+
+public class App {
+    public static void main(String[] args) {
+        System.out.println(Utils.isAllPositiveNumbers("12", "79"));
+        Utils.isAllPositiveNumbers(args);
```

Comment:
Why do we need the second call with args?

Reply:
repeat(to self) how main(args) works. removed